alfalfa_worker/step_sim/osm_model_advancer.py

```diff
@@ -409,9 +414,52 @@ def update_db_with_model_outputs(self):
                     "$set": {"rec.curVal": "n:%s" % output_value, "rec.curStatus": "s:ok",
                              "rec.cur": "m:"}}, False)
 
-    def update_sim_time_in_db(self):
+    def update_sim_time_in_mongo(self):
         """Placeholder for updating the datetime in Mongo to current simulation time"""
         output_time_string = "s:" + str(self.get_energyplus_datetime())
         self.ac.mongo_db_recs.update_one({"_id": self.site_id}, {
             "$set": {"rec.datetime": output_time_string, "rec.step": "n:" + str(self.ep.kStep),
                      "rec.simStatus": "s:Running"}}, False)
+
+    def write_outputs_to_influx(self):
+        """
+        Write output data to influx
+        :return:
+        """
+        json_body = []
+        base = {
+            "measurement": self.site_id,
+            "time": f"{self.get_energyplus_datetime()}",
+        }
+        response = False
+        for output_id in self.variables.get_output_ids():
```

Comment:
@corymosiman12 I don't think you are doing anything that hasn't already been done as we push current values to mongo, but I do wonder how much these get_foo methods are costing us. If ever we need to start performance optimizing this thing, the thing I would be looking at (right after moving current values into redis) would be these get index/id methods and the frequency that they are used. Not suggesting any revision now.

Reply:
yeah, agreed. some of them hit the DB, but most of them actually are just looking at hash maps (this one is just getting hash keys).